src/helpers/resolveValue.js

```diff
@@ -0,0 +1,7 @@
+export default function resolveValue(valueGetterFn, options = {}) {
```

Comment:
Since `options` is effectively optional here, can we have a default timeout? Similar to cypress's default timeout of 4000 - https://docs.cypress.io/guides/references/configuration.html#Timeouts